utils/proxy_pool.py

```
@@ -12,481 +12,246 @@
+
     Returns:
-        Configured ProxyPool instance
+        Configured RustProxyPool instance
     """
-    pool = ProxyPool(
+    rust_proxy_list = []
+    for proxy_config in proxy_list_config:
+        rust_proxy_dict = {}
+        if 'name' in proxy_config:
+            rust_proxy_dict['name'] = proxy_config['name']
+        if 'http' in proxy_config and proxy_config['http']:
```

Ruff (0.15.2)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)